tests/drivers/dma/chan_link_transfer/testcase.yaml

```diff
@@ -3,6 +3,8 @@ tests:
     min_ram: 16
     depends_on: dma
     tags: drivers dma
-    platform_allow: frdm_k64f mimxrt1060_evk mimxrt1064_evk mimxrt1160_evk_cm7 mimxrt1170_evk_cm7
+    platform_allow: |
+                    frdm_k64f mimxrt1060_evk mimxrt1064_evk mimxrt1160_evk_cm7 mimxrt1170_evk_cm7
+                    mimxrt1024_evk
```

Comment:
platform_allow supports lists, how about:
```
platform_allow:
  - frdm_k64f
  - mimxrt1060_evk
  - mimxrt1064_evk
  - mimxrt1160_evk_cm7
  - mimxrt1170_evk_cm7
  - mimxrt1024_evk
```

Reply:
Sure, I added this change.